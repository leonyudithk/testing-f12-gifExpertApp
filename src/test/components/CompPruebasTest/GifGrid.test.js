import React from 'react'
import { shallow } from "enzyme"
import { GifGrid } from '../../../components/GifGrid'


describe('pruebas al componente GifGrid', ()=>{
    test("debe mostrar todas las categorias", ()=>{
        //Arreglar
        const category  = '##11@@@'

        const wrapper = shallow( <GifGrid category={category}/>)

        //actuar
        expect(wrapper).toMatchSnapshot()
        expect(wrapper.find('GifGridItem').length).not.toBe(category.length)
        //expect(wrapper.find('GifGridItem').length).toBe(category.length)
    })
})