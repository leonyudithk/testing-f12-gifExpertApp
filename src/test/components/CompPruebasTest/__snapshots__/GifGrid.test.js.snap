// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pruebas al componente GifGrid debe mostrar todas las categorias 1`] = `
<Fragment>
  <h3
    className="animate__animated animate__fadeIn"
  >
     
    ##11@@@
     
  </h3>
  <p
    className="animate__animated animate__flash"
  >
    Loading
  </p>
  <div
    className="card-grid"
  />
</Fragment>
`;
